test/test_save_as_netcdf.py

```diff
@@ -30,7 +30,6 @@ def nc_has_variable(ds, var_name):
     if var_name in ds.variables:
         return True
     else:
-        print(var_name, " is not a variable in the Dataset")
```

Comment:
Yes... I am missing something! It tests the `pyugrid` saved data.

PS: I returned the print statement, but I think `py.test` verbose mode would give you the full `traceback` and that would not be needed.